DataFormats/TrackReco/src/classes_def.xml

```diff
@@ -1,11 +1,12 @@
 <lcgdict>
-  <class name="std::bitset<reco::TrackBase::algoSize>"/>
+  <class name="reco::TrackBase::AlgoMask"/>
   <class name="reco::HitPattern" ClassVersion="12">
       <version ClassVersion="12" checksum="3922863495"/>
       <version ClassVersion="11" checksum="1621684703"/>
   </class>
 
-  <class name="reco::TrackBase" ClassVersion="13">
+  <class name="reco::TrackBase" ClassVersion="15">
+   <version ClassVersion="15" checksum="3673246687"/>
    <version ClassVersion="14" checksum="3929365050"/>
    <version ClassVersion="13" checksum="1244921154"/>
```

Comment:
Slava,
Good catch.
Yes, it matters that the same version number is used for two different checksums.
I will need to fix this.  I may also need to change this file in 7_5_X.
Thank you,
Bill